===================================================================
RCS file: /cvsroot/pixdev/pixdev-stable/include/nxtminput.h,v
retrieving revision 1.1.1.1
retrieving revision 1.2
diff -u -r1.1.1.1 -r1.2
--- pixdev-stable/include/nxtminput.h	2003/10/10 14:51:12	1.1.1.1
+++ pixdev-stable/include/nxtminput.h	2003/12/06 06:22:41	1.2
@@ -54,7 +54,8 @@
 #define			MAX_UNITS		4	// Maximum number of units (HH:MM:SS:[am/pm])
 
 //const char                            *dayzn[2] = {{"AM"}, {"PM"}};                   // Array of either am or pm
-const short max_val[3] = { {23}, {59}, {59} };	// Max values per unit (23 = hours, 59 = min/sec)
+const short max_val[3] = { 23, 59, 59 };
+// Max values per unit (23 = hours, 59 = min/sec)
 
 
 /* Forward declarations */
